python/openassetio/test/manager/harness.py

```diff
@@ -30,8 +30,14 @@
 
 from . import _implementation
 
+# Hoist our locale from our private implementation (we don't declare
+# it here as otherwise we'd end up with circular imports trying
+# to use it in the harness loader).
+from ._implementation import ManagerTestHarnessLocale
```

Comment:
[question] It looks like the only reason this is hoisted and exported is for the unit tests? I don't think it's so bad if the unit tests access `_implementation`.  Is there another reason?  Do we expect end users will need access to the class via `harness`, perhaps?

Reply:
Maybe the `ManagerTestHarnessLocale` should be defined in the new `specifications`? Not sure.

Reply:
The `specifications` module is perfect - great idea. It needs to be publicly visible as any new specification introduced by a host must be available to the manager/etc... so they can test against it/know its properties/etc...

Reply:
Not quite what I meant, but this seems reasonable to me.